presto-benchto-benchmarks/src/test/resources/sql/presto/tpcds/q05.plan.txt

```diff
@@ -1,57 +1,56 @@
-        final aggregation over (channel$gid, groupid, id$gid)
-            local exchange (REPARTITION, HASH, ["channel$gid", "groupid", "id$gid"])
-                remote exchange (REPARTITION, HASH, ["channel$gid", "groupid", "id$gid"])
-                    partial aggregation over (channel$gid, groupid, id$gid)
-                        local exchange (REPARTITION, ROUND_ROBIN, [])
-                            final aggregation over (s_store_id)
-                                local exchange (GATHER, SINGLE, [])
-                                    remote exchange (REPARTITION, HASH, ["s_store_id"])
-                                        partial aggregation over (s_store_id)
+remote exchange (GATHER, SINGLE, [])
+    final aggregation over (channel$gid, groupid, id$gid)
+        local exchange (REPARTITION, HASH, ["channel$gid", "groupid", "id$gid"])
+            remote exchange (REPARTITION, HASH, ["channel$gid", "groupid", "id$gid"])
+                partial aggregation over (channel$gid, groupid, id$gid)
+                    local exchange (REPARTITION, ROUND_ROBIN, [])
+                        final aggregation over (s_store_id)
+                            local exchange (GATHER, SINGLE, [])
+                                remote exchange (REPARTITION, HASH, ["s_store_id"])
+                                    partial aggregation over (s_store_id)
+                                        join (INNER, REPLICATED):
                                             join (INNER, REPLICATED):
-                                                join (INNER, REPLICATED):
-                                                    remote exchange (REPARTITION, ROUND_ROBIN, [])
-                                                        scan store_sales
-                                                        scan store_returns
-                                                    local exchange (GATHER, SINGLE, [])
-                                                        remote exchange (REPLICATE, BROADCAST, [])
-                                                            scan date_dim
+                                                remote exchange (REPARTITION, ROUND_ROBIN, [])
+                                                    scan store_sales
+                                                    scan store_returns
                                                 local exchange (GATHER, SINGLE, [])
                                                     remote exchange (REPLICATE, BROADCAST, [])
-                                                        scan store
-                            final aggregation over (cp_catalog_page_id)
-                                local exchange (GATHER, SINGLE, [])
-                                    remote exchange (REPARTITION, HASH, ["cp_catalog_page_id"])
-                                        partial aggregation over (cp_catalog_page_id)
+                                                        scan date_dim
```

Comment:
tables are joined here in different order.

Do you have any possibility to benchmark this?